operator_ui/src/pages/JobsIndex/DirectRequestRow.tsx

```diff
@@ -33,41 +33,41 @@ interface Props extends WithStyles<typeof styles> {
   job: DirectRequest
 }
 
-export const DirectRequestRow = withStyles(styles)(
-  ({ job, classes }: Props) => {
-    return (
-      <TableRow style={{ transform: 'scale(1)' }} hover>
-        <TableCell className={classes.cell} component="th" scope="row">
-          <Link className={classes.link} href={`/jobs/${job.id}`}>
-            {job.attributes.name || job.id}
-            {job.attributes.name && (
-              <>
-                <br />
-                <Typography
-                  variant="subtitle2"
-                  color="textSecondary"
-                  component="span"
-                >
-                  {job.id}
-                </Typography>
-              </>
-            )}
-          </Link>
-        </TableCell>
-        <TableCell>
-          <Typography variant="body1">
-            <TimeAgo tooltip>{job.attributes.createdAt}</TimeAgo>
-          </Typography>
-        </TableCell>
-        <TableCell>
-          <Typography variant="body1">Direct request</Typography>
-        </TableCell>
-        <TableCell>
-          <Typography variant="body1">
-            {formatInitiators(job.attributes.initiators)}
-          </Typography>
-        </TableCell>
-      </TableRow>
-    )
-  },
-)
+const DirectRequestRow = withStyles(styles)(({ job, classes }: Props) => {
+  return (
+    <TableRow style={{ transform: 'scale(1)' }} hover>
+      <TableCell className={classes.cell} component="th" scope="row">
+        <Link className={classes.link} href={`/jobs/${job.id}`}>
+          {job.attributes.name || job.id}
+          {job.attributes.name && (
+            <>
+              <br />
+              <Typography
+                variant="subtitle2"
+                color="textSecondary"
+                component="span"
+              >
+                {job.id}
+              </Typography>
+            </>
+          )}
+        </Link>
+      </TableCell>
+      <TableCell>
+        <Typography variant="body1">
+          <TimeAgo tooltip>{job.attributes.createdAt}</TimeAgo>
+        </Typography>
+      </TableCell>
+      <TableCell>
+        <Typography variant="body1">Direct request</Typography>
+      </TableCell>
+      <TableCell>
+        <Typography variant="body1">
+          {formatInitiators(job.attributes.initiators)}
+        </Typography>
+      </TableCell>
+    </TableRow>
+  )
+})
+
+export default DirectRequestRow
```

Comment:
Named export. I haven't had a chance to, but if you have the time it would be great to add https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-default-export.md so comments like this would be redundant.

Reply:
OK I can add a ticket and do this in a separate commit